draft-ietf-ice-pac.xml

```diff
@@ -212,17 +213,19 @@
         the best position to determine what is reasonable for its scenario.
       </t>
       <t>
-        While the timer is running, the ICE agent MUST NOT set the state of
-        a checklist to Failed, even if the checklist has no pairs left to check.
-        As a result, the ICE agent will not remove any data streams or set the
-        state of the ICE session to Failed as long as the timer is running.
+        While the timer is still running, the ICE agent MUST NOT update a checklist state
+        from Running to Failed, even if there are no pairs left in the checklist to check.
+        As a result, the ICE agent will not remove any data streams or set the state of the
+        ICE session to Failed as long as the timer is running. As a result, the ICE agent will
+        not remove any data streams or set the state of the ICE session to Failed as long as
+        the timer is running.
       </t>
       <t>
         When the timer eventually elapses, the ICE agent MUST resume typical
-        ICE processing, including setting any checklists containing only Failed
-        pairs to the Failed state, as usual, and handling any consequences as
-        indicated in <xref target="RFC8445" />, Section 8.1.2. Naturally, if
-        there are no such checklists, no action is necessary.
+        ICE processing, including setting the state of a checklist to Failed if there
```

Comment:
"the state of any checklists to Failed if they have no pairs left to check"

Reply:
Will fix as suggested.